core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/resources/empty-CamelFileWithRoutes.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd        http://camel.apache.org/schema/spring http://camel.apache.org/schema/spring/camel-spring.xsd">
+	<routes xmlns="http://camel.apache.org/schema/spring" id="context1">
+	 <route id="route3">
+	 </route>
+  </routes>
+</beans>
```

Comment:
missing new line at end

Reply:
why does it need a new line at end? To have Linux recognizing the file as a Text file?

Reply:
thats what github complains about when you view your code

Reply:
having newlines at end of line is helping to have easier when using command-line and git diff
https://mrkiffie.com/2016/newlines-and-version-control/

Reply:
new line added